public/scripts/main.js

```diff
@@ -127,12 +129,23 @@ const gameHandler = (function gameIIFE() {
     })
   }
 
+  function displayRole(role) {
+    document.querySelector('#image').src = role.image
+    document.querySelector('#name').textContent = role.name
+    document.querySelector('#description').textContent = role.description
+  }
+
   // Transitions
   function onEnter() {
     socket.on('message', appendIncomingMessage)
     input.addEventListener('input', validateInput)
     input.addEventListener('keyup', enterKeyPressed(sendMessage))
     game.hidden = false
+    const STARTING_REQUIRED_WORD_COUNT = 10
+    requiredWords.addWords(STARTING_REQUIRED_WORD_COUNT)
+    const role = roles.GetHumanRole()
+    // const role = roles.GetAIRole()
```

Comment:
Could you add the server code to select roles as well, or do you want that to be a separate bit of work? I feel like it should be fairly trivial to add, but maybe I'm wrong